requirements_s3.txt

```
@@ -2,7 +2,7 @@ boto3==1.20.17
 botocore==1.23.17
 jmespath==0.10.0
 python-dateutil==2.8.2
-s3path==0.3.2
+s3path>=0.3.3
```

```suggestion
s3path==0.3.3
```
Here we pin to say this is the versions CI has tested.